Release `1.2.0`

robbot006 opened the pull request:
This is a draft of changelog for the `1.2.0` release.
Here you can find all related translations changes: [1.1...master](https://github.com/icecore2/Flarum-hebrew/compare/1.1...master).

> ### ⚠️ Do not merge this pull request manually ⚠️
> 
> You should [approve](https://github.com/rob006-software/flarum-translations/wiki/How-to-approve-release-pull-request) this pull request instead. After approving, I will automatically update it (release date needs to be adjusted), merge, and tag a new release.

<details>
<summary>Show release notes preview</summary>

## 1.2.0

<!-- release-notes-begin -->
**General changes**:

* Updated Flarum core translations.
* Updated validation translations.


**Added support for new extensions**:

* [`acpl/flarum-lscache`](https://github.com/android-com-pl/flarum-lscache)
* [`afrux/forum-widgets-core`](https://github.com/afrux/forum-widgets-core)
* [`afrux/news-widget`](https://github.com/afrux/news-widget)
* [`afrux/online-users-widget`](https://github.com/afrux/online-users-widget)
* [`afrux/top-posters-widget`](https://github.com/afrux/top-posters-widget)
* [`antoinefr/flarum-ext-online`](https://github.com/AntoineFr/flarum-ext-online)
* [`askvortsov/flarum-article-series`](https://github.com/askvortsov1/flarum-article-series)
* [`askvortsov/flarum-categories`](https://github.com/askvortsov1/flarum-categories)
* [`askvortsov/flarum-checklist`](https://github.com/askvortsov1/flarum-checklist)
* [`askvortsov/flarum-help-tags`](https://github.com/askvortsov1/flarum-help-tags)
* [`blomstra/mark-unread`](https://github.com/blomstra/flarum-ext-mark-unread)
* [`blomstra/notification-deleter`](https://github.com/blomstra/flarum-ext-notification-deleter)
* [`blomstra/realtime`](https://extiverse.com/extension/blomstra/realtime)
* [`blomstra/welcome-login`](https://github.com/blomstra/flarum-ext-welcome-login)
* [`flarum/statistics`](https://github.com/flarum/statistics)
* [`fof/prevent-necrobumping`](https://github.com/FriendsOfFlarum/prevent-necrobumping)
* [`fof/socialprofile`](https://github.com/FriendsOfFlarum/socialprofile)
* [`kilowhat/flarum-ext-audit-free`](https://github.com/kilowhat/flarum-ext-audit-free)
* [`kilowhat/flarum-ext-audit-pro`](https://extiverse.com/extension/kilowhat/flarum-ext-audit-pro)
* [`zerosonesfun/expired-posts`](https://github.com/zerosonesfun/expired-posts)


All changes: [1.1...1.2.0](https://github.com/icecore2/Flarum-hebrew/compare/1.1...1.2.0).



<!-- release-notes-end -->

</details>

robbot006 committed:
Update changelog.

rob006 commented:
@icecore2 Just as a note, this is an automated PR generated by bot. The idea is to simplify release process for language pack maintainers - all you need is to approve this pull request and bot will handle the rest. You can read more about the process at forum: https://discuss.flarum.org/d/20807-simplify-translation-process-with-weblate/76 . If you don't want to use this feature, let me know.

Also, please take a look at https://github.com/rob006-software/flarum-translations/issues/924. Because this is private repository I can't setup webhooks correctly, so bot may be quite slow to respond. Migrating repository under `flraum-lang` org would help with that.

robbot006 committed:
Update changelog.

robbot006 commented:
Success! Now you can announce new release on [forum](https://discuss.flarum.org/t/languages):

~~~markdown
## Version [`1.2.0`](https://github.com/icecore2/Flarum-hebrew/releases/tag/1.2.0)

**General changes**:

* Updated Flarum core translations.
* Updated validation translations.


**Added support for new extensions**:

* [`acpl/flarum-lscache`](https://github.com/android-com-pl/flarum-lscache)
* [`afrux/forum-widgets-core`](https://github.com/afrux/forum-widgets-core)
* [`afrux/news-widget`](https://github.com/afrux/news-widget)
* [`afrux/online-users-widget`](https://github.com/afrux/online-users-widget)
* [`afrux/top-posters-widget`](https://github.com/afrux/top-posters-widget)
* [`antoinefr/flarum-ext-online`](https://github.com/AntoineFr/flarum-ext-online)
* [`askvortsov/flarum-article-series`](https://github.com/askvortsov1/flarum-article-series)
* [`askvortsov/flarum-categories`](https://github.com/askvortsov1/flarum-categories)
* [`askvortsov/flarum-checklist`](https://github.com/askvortsov1/flarum-checklist)
* [`askvortsov/flarum-help-tags`](https://github.com/askvortsov1/flarum-help-tags)
* [`blomstra/mark-unread`](https://github.com/blomstra/flarum-ext-mark-unread)
* [`blomstra/notification-deleter`](https://github.com/blomstra/flarum-ext-notification-deleter)
* [`blomstra/realtime`](https://extiverse.com/extension/blomstra/realtime)
* [`blomstra/welcome-login`](https://github.com/blomstra/flarum-ext-welcome-login)
* [`flarum/statistics`](https://github.com/flarum/statistics)
* [`fof/prevent-necrobumping`](https://github.com/FriendsOfFlarum/prevent-necrobumping)
* [`fof/socialprofile`](https://github.com/FriendsOfFlarum/socialprofile)
* [`kilowhat/flarum-ext-audit-free`](https://github.com/kilowhat/flarum-ext-audit-free)
* [`kilowhat/flarum-ext-audit-pro`](https://extiverse.com/extension/kilowhat/flarum-ext-audit-pro)
* [`zerosonesfun/expired-posts`](https://github.com/zerosonesfun/expired-posts)


All changes: [1.1...1.2.0](https://github.com/icecore2/Flarum-hebrew/compare/1.1...1.2.0).

In order to update run these commands:

```console
composer update flabrew/flabrew
php flarum cache:clear
```

~~~
